packages/x-data-grid/src/models/api/gridCallbackDetails.ts

```diff
@@ -8,4 +9,8 @@ export interface GridCallbackDetails<K extends keyof GridControlledStateReasonLo
    * The reason for this callback to have been called.
    */
   reason?: GridControlledStateReasonLookup[K];
+  /**
+   * GridApi that let you manipulate the grid.
+   */
+  api: GridApiCommunity;
```

Comment:
Sidenote but I don't like that we don't have a strict convention of either always `apiRef` or always `api`. Sometimes we need to do awkward conversions like `{ current: api }` to create a new "ref".